### ISSUE TITLE
Locked on webtoon mode on mistaged titles

### ISSUE THREAD

**Issue by D-Brox** [CarlosEsco/Neko#1233]
### Steps to reproduce

1. Open a title that has been mistaged as `Long Strip`, such as [Romantic Killer](https://mangadex.org/title/b1b20403-c861-4865-8cbe-3d3b39f176c5), or that changed it's reading mode mid-publication, such as [Hide Your Tears, You Crybaby](https://mangadex.org/title/c3129d25-fe12-4b0a-b415-6ea0d2fba662)
2. Try to switch the reading mode

### Expected behavior

Being able to change reading mode.

(There could even be a "are you sure?" button)

### Actual behavior

Reading mode selection is grayed out, leaving the mode locked in `Long Strip`

### Neko version

2.11.6

### Android version

Android 12

### Device

Xiaomi Poco X4 Pro 5G

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
For wrong tags, report them on dex.  As for [Hide Your Tears, You Crybaby](https://mangadex.org/title/c3129d25-fe12-4b0a-b415-6ea0d2fba662) I am not seeing how its not a longstrip

**Comment by D-Brox** [comment]
> For wrong tags, report them on dex.  As for [Hide Your Tears, You Crybaby](https://mangadex.org/title/c3129d25-fe12-4b0a-b415-6ea0d2fba662) I am not seeing how its not a longstrip

@CarlosEsco , it started as longstrip, but from ch3 onwards, it changed to horizontal, manga orientation. That's what I meant with "changed it's reading mode mid-publication".

For example, this pages are clearly non-contiguous, but I can't do anything because reading mode is locked:
![Screenshot_2022-11-15-22-33-40-882_tachiyomi.mangadex.jpg](https://user-images.githubusercontent.com/65723952/202061624-3b196201-1da6-42dd-bb94-59c0210cb08d.jpg)

**Comment by CaidenBlock** [comment]
I just came across this and [this seems to have been mentioned before on the tachi discord](https://discord.com/channels/349436576037732353/569823838305648640/941315649383325716) with another series as stupid mid-series switches from webtoon to pages reading has happened with a few series, unlocking the reading mode selector would help

tbh, I don't know how many people would even bother with changing the per-series reading mode away from the default, it makes me think that the annoyance of some series is greater than the freedom to allow users to do what they want- even if it's stupid most of the time.

**Comment by D-Brox** [comment]
> I just came across this and [this seems to have been mentioned before on the tachi discord](https://discord.com/channels/349436576037732353/569823838305648640/941315649383325716) with another series.
> 
> As stupid mid-series switches from webtoon to pages reading has happened with a few series, unlocking the reading mode selector would help
> 
> tbh, I don't know how many people would even bother with changing the per-series reading mode away from the default
> 
> It makes me think that the annoyance of some series is greater than the freedom to allow users to do what they want- even if it's stupid most of the time.

It's an edge case, but there are a few other examples. I'd suggest allowing to change from longstrip, but only after prompting for confirmation from the user. It still should default to longstrip when there is the tag, tho.

Screenshots of the messages on discord:
![Screenshot_2023-04-02-19-32-34-102_com.aliucord.png](https://user-images.githubusercontent.com/65723952/229382658-e3f70bf8-2c54-4057-846f-c84fd299536b.png)

![Screenshot_20220210-042651_Neko.jpg](https://user-images.githubusercontent.com/65723952/229382657-7a241d2c-8317-49b3-b8af-6ab3b7207f9a.jpg)

**Comment by CarlosEsco** [comment]
I understand this should probably be changed for dumb series like this. But also the dex tag should be updated when a series changes like this
